creditnote.go

```diff
@@ -53,6 +53,20 @@ type CreditNoteListParams struct {
 	Invoice    *string `form:"invoice"`
 }
 
+// CreditNotePreviewParams is the set of parameters that can be used when previewing a credit note.
+// For more details see https://stripe.com/docs/api/credit_notes/create, https://stripe.com/docs/api/credit_notes/update.
+type CreditNotePreviewParams struct {
+	Params           `form:"*"`
+	Amount           *int64  `form:"amount"`
+	CreditAmount     *int64  `form:"credit_amount"`
+	Invoice          *string `form:"invoice"`
+	Memo             *string `form:"memo"`
+	OutOfBankdAmount *int64  `form:"out_of_band_amount"`
```

Comment:
```suggestion
	OutOfBandAmount *int64  `form:"out_of_band_amount"`
```

Reply:
sadness, it's a copy-paste of one above....